docs/source/development-environment.rst

```diff
@@ -0,0 +1,141 @@
+Zipline Development Guidelines
+==============================
+
+Environment Setup
+-----------------
+
+First, you'll need to clone zipline by running:
+
+.. code-block:: bash
+
+   git clone git@github.com:quantopian/zipline.git
+
+Then check out to a new branch where you can make your changes:
+
+.. code-block:: bash
+		
+   git checkout -b some-short-descriptive-name
+
+The following section assumes you already have virtualenvwrapper and pip installed on your system. If you don't already have them, you'll need some C library dependencies. On Linux you can run:
+
+.. code-block:: bash
+
+   # on linux
+   $ sudo apt-get install libopenblas-dev liblapack-dev gfortran
+
+   
+   $ wget http://prdownloads.sourceforge.net/ta-lib/ta-lib-0.4.0-src.tar.gz
```

Comment:
IMO, we should just make TA-Lib an optional dependency.  Nothing in zipline actually needs it; it's just required to run the tests.